### ISSUE TITLE
Czech characters looks correct when editing, but not in view mode

### ISSUE THREAD

**Issue by vlcinsky** [structurizr/structurizr#57]
Might be related to issue #25

OS: Linux Debian 11 and Windows
Browser: Firefox 91.6esr and Chromium Version 98.0.4758.80 (Official Build) built on Debian 11.2, running on Debian 11.2 (64-bit)

Using Strucrurizr in trial mode.

Workspace: https://structurizr.com/workspace/72250/diagram-editor#00-land-base-show

When I open my workspace and view the first diagram, it renders special Czech characters very badly:
![obrazek](https://user-images.githubusercontent.com/635911/153586744-8b2c0f59-86d8-44d4-a33c-4dde6229d6a2.png)
When I start editing the diagram, it looks correct:
![obrazek](https://user-images.githubusercontent.com/635911/153586869-ea26fc80-743f-4480-a36e-ec5bd50cb522.png)

When I try to export diagrams into HTML page (this was my motivation to subscribe), the page looks even worse.

When I edit the same document using Structurizr Lite, things looks nice (but there is no "view only" mode).

**Comment by simonbrowndotje** [comment]
This should be working once again, although you may need to push your DSL (using the CLI or DSL editor page) to force regeneration of the workspace content.

**Comment by vlcinsky** [comment]
Thanks @simonbrowndotje for very quick reaction.

I can confirm, things are working well now.

Just for record:

1. My workspace was originally populated by importing existing workspace.json file
2. it had the issue reported above: read only view had broken characters, edit view looked well
3. as advised, I used structurizr cli to push the workspace.json to the workspace
4. as a result - all views (read only as well the view for editing) shows the text properly

To make sure, the problem is resolved for newly created workspaces with uploaded workspace.json I have tried in another space to upload (via web UI) the workspace.json which contains model with many special Czech characters and all works well.

Thanks a lot.
